AdyenComponents/OnlineBanking/OnlineBankingComponent.swift

```diff
@@ -0,0 +1,132 @@
+import UIKit
+
+/// A component that provides a form for Online Banking payment.
+public final class OnlineBankingComponent: PaymentComponent,
+                                           PresentableComponent,
+                                           LoadingComponent,
+                                           PaymentAware {
+
+    /// Configuration for Online Banking Component.
+    public typealias Configuration = BasicComponentConfiguration
+
+    /// The context object for this component.
+    @_spi(AdyenInternal)
+    public var context: AdyenContext
+
+    public var paymentMethod: PaymentMethod { onlineBankingPaymentMethod }
+
+    public weak var delegate: PaymentComponentDelegate?
+
+    public lazy var viewController: UIViewController = SecuredViewController(child: formViewController,
+                                                                             style: configuration.style)
+
+    public var requiresModalPresentation: Bool = true
+
+    /// Component's configuration
+    public var configuration: Configuration
+
+    private let onlineBankingPaymentMethod: OnlineBankingPaymentMethod
+
+    private var selectedIssuer: Issuer
+
+    // MARK: - Items
+
+    private var termsAndConditionsLink: String {
+        return paymentMethod.type == .onlineBankingCZ ? "https://static.payu.com/sites/terms/files/payu_privacy_policy_cs.pdf" : "https://static.payu.com/sites/terms/files/payu_privacy_policy_sk.pdf"
+    }
+
+    /// The terms and condition message item.
+    // swiftlint:disable line_length
+    internal lazy var termsAndConditionsLabelItem: FormAttributedLabelItem = .init(termsAndConditionsText: localizedString(.onlineBankingTermsAndConditions, configuration.localizationParameters),
+                                                                      link: termsAndConditionsLink,
+                                                                      style: configuration.style.footnoteLabel,
+                                                                      linkTextStyle: configuration.style.linkTextLabel,
+                                                                      identifier: ViewIdentifierBuilder.build(scopeInstance: "AdyenDropIn.OnlineBankingComponent", postfix: "OnlineBankingTAndCLabel"))
+
+    /// The continue button item.
+    internal lazy var continueButton: FormButtonItem = {
+        let item = FormButtonItem(style: configuration.style.mainButtonItem)
+        item.identifier = ViewIdentifierBuilder.build(scopeInstance: "AdyenDropIn.OnlineBankingComponent", postfix: "continueButtonItem")
+        item.title = localizedString(.continueTitle, configuration.localizationParameters)
+        item.buttonSelectionHandler = { [weak self] in
+            self?.didSelectContinueButton()
+        }
+        return item
+    }()
+
+    /// The Issuer List item.
+    internal lazy var issuerListItem: FormIssuersPickerItem = {
+        let defaultIssuer = onlineBankingPaymentMethod.issuers[0]
```

Comment:
Better to be cautious of bounds when accessing arrays. You can use `safeIndex` subscript